### ISSUE TITLE
npm package 0.1.1 does not include change to allow node 5

### ISSUE THREAD

**Issue by bdefore** [karma-runner/karma-cli#17]
npm installing this version with node 5 gives the warning:

```
npm WARN engine karma-cli@0.1.1: wanted: {"node":"0.10 || 0.12 || 4"} (current: {"node":"5.2.0","npm":"3.3.12"})
```

can we get a new publish?

**Comment by frankleng** [comment]
:+1:

**Comment by Dignifiedquire** [comment]
Fixed in 0.1.2
